src/index.ts

```diff
@@ -101,11 +101,15 @@ export type {
   MultiSelectItem,
   SelectedItems,
 } from "./components/MultiSelect/MultiSelect";
-// The code associated with the MultiSelectGroup component will not be exported
-// until the refactor of the v2 version of the MultiSelectGroup component has
-// been completed.
+/** The code associated with the MultiSelectGroup component will not be exported
+ * until the refactor of the v2 version of the MultiSelectGroup component has
+ * been completed.
+ * */
 // export { default as MultiSelectGroup } from "./components/MultiSelectGroup/MultiSelectGroup";
 // export type { MultiSelectGroupProps } from "./components/MultiSelectGroup/MultiSelectGroup";
+/** The code associated with the NewsletterSignup component will not be exported
+ * until the refactor of the component has been completed.
+ * */
 export { default as NewsletterSignup } from "./components/NewsletterSignup/NewsletterSignup";
```

Comment:
Make sure to comment the component export.

Reply:
Oops.